vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -148,7 +148,7 @@ type internal FSharpAddOpenCodeFixProvider
         asyncMaybe {
             let! options = projectInfoManager.TryGetOptionsForEditingDocumentOrProject context.Document
             let! sourceText = context.Document.GetTextAsync(context.CancellationToken)
-            let! parsedInput, checkResults = checker.ParseAndCheckDocument(context.Document, options, sourceText)
+            let! _, parsedInput, checkResults = checker.ParseAndCheckDocument(context.Document, options, sourceText)
```

Comment:
I believe multiple checkings will interleave:

* A checks the cache - None
* B checks the cache - None
* A starts queuing 50 ms chunks
* B starts queuing 50 ms chunks

A and B have no idea of each other, so checking is done twice, in parallel. 

Would check the cache each 50 ms be OK? It should stops duplicated work easily.